packages/functional-tests/src/server.ts

```diff
@@ -15,6 +15,7 @@ process.on('uncaughtException', (err) => console.log('uncaughtException', err));
 process.on('unhandledRejection', (reason) => console.log('unhandledRejection', reason));
 
 MRE.log.enable('app');
+MRE.log.enable('client-trace');

```

Comment:
>client [](start = 16, length = 6)

Let's enable this by default (done in log.ts checkInitialize method). By the way, 'app' is enabled by default now, no longer necessary to enable it here.